controllers/disruption_controller.go

```diff
@@ -140,7 +140,7 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	// check whether the object is being deleted or not
-	if !instance.ObjectMeta.DeletionTimestamp.IsZero() {
+	if !instance.DeletionTimestamp.IsZero() {
```

Comment:
Afaict, `instance.DeletionTimestamp` is the exact same as `instance.ObjectMeta.DeletionTimestamp`, yes?

Reply:
We don't favor one or the other, afaik. I was more curious if you knew a reason that the shorter form was preferable. If they're identical but one is more concise, I'm definitely in support of the shorter option

Reply:
Not really, I favoured it because it was the shorter option indeed.
The main difference is the short option due to go, is that we can have a meaning that changes:
* as long as there is no field in Disruption called `DeletionTimestamp` it's linking to `ObjectMeta.DeletionTimestamp`
* if at any time we have a second embedded struct containing this field, it will lead to a compile time error 
```text
ambiguous selector instance.DeletionTimestampcompiler[AmbiguousSelector](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#AmbiguousSelector)
```
* if we create the same field `DeletionTimestamp` we would have a compile error due to missing method

The only risk in using the shortest would hence be if we introduce the exact same field with the exact same type. I would say it's a rather edge case we might not be interested in preventing.